airflow/models/baseoperator.py

```diff
@@ -221,7 +221,8 @@ class derived from this one results in the creation of a task object,
 
     :param task_id: a unique, meaningful id for the task
     :type task_id: str
-    :param owner: the owner of the task, using the unix username is recommended
+    :param owner: str, the owner of the task. Using a meaningful description
```

Comment:
```suggestion
    :param owner: the owner of the task. Using a meaningful description
```
I think the 'str' here is not necessary due to the `type` description below